analysis/stat_tests.py

```diff
@@ -179,3 +166,50 @@ def friedman_posthoc_tests(experiment_pivot_df):
     posthoc_tests['conover'] = sp.posthoc_conover_friedman(experiment_pivot_df)
     posthoc_tests['nemenyi'] = sp.posthoc_nemenyi_friedman(experiment_pivot_df)
     return posthoc_tests
+
+
+def a12(measurements_x, measurements_y, alternative=None):
```

Comment:
Do we still need the alternative parameter?

Reply:
nope, I should have removed this as well.